README.md

```diff
@@ -9,10 +9,13 @@ is a cross-browser `Blob` implementation that solves this.
 Supported browsers
 ------------------
 
-canvas-toBlob.js has [the same browser support as FileSaver.js][3].
+| Feature                 | Chrome | Firefox (Gecko) | Internet Explorer | Opera | Safari         |
+|-------------------------|--------|-----------------|-------------------|-------|----------------|
+| Basic support           | 50     | 19              | 10 -ms            | 37    | polyfill [[1][3]] |
```

Comment:
The "\[1\]" link is confusing. It seems to imply that it would be a link to the polyfill, but it's a link to the issue for Safari getting toBlob. Maybe you should reword or rearrange it a little.